examples/duoswap-core/Makefile

```diff
@@ -0,0 +1,16 @@
+all: build
+
+.PHONY: clean
+clean:
+	rm -rf ../ext-duoswap-core
+
+checkout:
+	if test -d ../ext-duoswap-core; then echo "Repo already cloned"; else git clone https://github.com/reach-sh/duoswap-core ../ext-duoswap-core; fi
+
+.PHONY: build
+build: checkout
+	cd ../ext-duoswap-core && ../one.sh build
+
+.PHONY: run
+run:
+	echo "external code will not be ran"
```

Comment:
What you have is valuable... we're ensuring that we don't break their code's compilation, but it is a little unsatisfying that we don't run it either. You're scared it will take too long? You're scared that they will blow us up? Can't think of a good "protocol" to tell it to run?

Reply:
I was thinking it could either be long, interactive, or have no capability to run (i.e. just a reach library). But, I could have the script generate run commands. Then, it's up to the developer adding the test to check whether it makes sense and adjust if necessary?

Reply:
Yea, those all make sense. In the case of duoswap, do you think it is worth running?

Reply:
Since it already runs in its own CI, I think having `reach-lang` check it can compile without long verification is fine. No strong feelings though